src/Kubernetes/KubernetesApiExtensions.cs

```diff
@@ -503,12 +503,37 @@ public static V1Job WithBillingId(this V1Job job, string billingId)
             });
         }
 
+        /// <summary>
+        /// Adds a policy failure action and exit codes to the job.
+        /// </summary>
+        /// <param name="job">The job object to modify.</param>
+        /// <param name="actionExitCodeId">A list with all action names and corresponding exit code number.</param>
```

Comment:
remember to fix these warnings re docstrings

Reply:
Oh I missed it and didn't even looked at the warnings 🤦‍♂️ 
I've updated it now, thank you for pointing that out 🙂